apps/web/src/hooks/useFindSessionQuery.ts

```diff
@@ -0,0 +1,37 @@
+import { $SessionWithUser } from '@opendatacapture/schemas/session';
+import type { SessionWithUserQueryParams } from '@opendatacapture/schemas/session';
+import { useQuery } from '@tanstack/react-query';
+import axios from 'axios';
+
+//Change this query to into a hook method and name it useFindSessionQuery
+
+//Change the api call to have an include tag which includes the username from users
+
+//Change the return type to
+
+type UseSessionOptions = {
+  enabled?: boolean;
+  params: SessionWithUserQueryParams;
+};
+
+export const useFindSessionQuery = (
```

Comment:
This should be named `useSessionQuery`, consistent with the other query hooks. The same goes for the user query.